packages/cactus-plugin-satp-hermes/src/test/typescript/unit/config-validating-functions/validate-key-pair-json.test.ts

```diff
@@ -0,0 +1,59 @@
+import "jest-extended";
+import { validateSatpKeyPairJSON } from "../../../../main/typescript/services/validation/config-validating-functions/validate-key-pair-json";
+import { KeyPairJSON } from "../../../../main/typescript/services/validation/config-validating-functions/validate-key-pair-json";
+
+describe("validateKeyPairJson", () => {
+  it("should pass with a valid key pair", () => {
+    expect(() =>
+      validateSatpKeyPairJSON({
+        configValue: {
+          privateKey: "0xabc123",
```

Comment:
This is not a valid key; thus, an error should be thrown in this circumstance. Please add a check in the code to only accept valid keys